src/main/cpp/VideoState.h

```diff
@@ -101,6 +108,39 @@ enum ShowMode {
 	SHOW_MODE_NB
 };
 
+// Note will have to divide by 1000 to get the float value of the rate 
+enum Rates {
+	X1D32,
+	X1D16,
+	X1D8,
+	X1D4,
+	X1D2,
+	X1,
+	X2,
+	X4,
+	X8,
+	X16,
+	X32,
+};
+static const struct RatesEntry {
+	enum Rates rate;
+	float clock_speed;
+	float pts_speed;
```

Comment:
Just compute pts_speed by 1/clock_speed rather than putting it here

Reply:
Changing the presentation time could be done through RatesEntry.command when using AVFilter and RatesEntry.pts_speed. In the case of pts_speed we will have to change the presentation time of each frame before the decoding process, unfortunately, this is not working as expected (I removed it from from the code). This was for clarity so we can refer to this variable as the speed of the presentation rather than 1/clock_speed which could be confusing.